### ISSUE TITLE
[Android] Switch other icon will enabled other icon.

### ISSUE THREAD

**Issue by johnnyjong823** [DevsOnFlutter/flutter_live_icon#4]
Hi Divshekhar

**Describe the bug**
After implementing the example, when you click to switch DarkTheme’s Icon,
The Icon of MainActivity will disappear, but the Icon of LightTheme and DarkTheme will exist together.

**To Reproduce**
1. Follow the implementation steps
2. Switch DarkTheme's Icon
3. See error

**Screenshots**
1.This MainActivity Icon.
<img src="https://user-images.githubusercontent.com/17687685/148322618-81384640-af2a-4238-9b5e-b5cfd1f9e922.png" alt="" width="300">
2.Click the DarkTheme's Button.
<img src="https://user-images.githubusercontent.com/17687685/148322691-7fcf5c41-cd4c-408c-9a06-2d3d3eb87c3c.png" alt="" width="300">
3.Switch Back to MainActivity.
<img src="https://user-images.githubusercontent.com/17687685/148322742-0e1732a9-6e13-4a2e-8dc8-d938e5fd477f.png" alt="" width="300">
Three Icon is enabled.😂😂

Solution

Search for keywords in the project: "DONT_KILL_APP"
Find the MethodCallImplementation.Java file

Find this line.
`pm.setComponentEnabledSetting(cn, componentState, PackageManager.DONT_KILL_APP);`

Add this code above it.
``` Java
else{
     componentState = PackageManager.COMPONENT_ENABLED_STATE_DISABLED;
}
```

It Work.

BR
Johnny

**Comment by divshekhar** [comment]
Hi @johnnyjong823, Thank you for reporting the bug and offering a fix. It would be great if you could prepare a PR with the solution. Please let me know whether you want this issue assigned to you.

**Comment by johnnyjong823** [comment]
Hi @divshekhar 
I send the PR to you.
You can check and merge.

If the correction is wrong, you can tell me.

BR
Johnny

**Comment by divshekhar** [comment]
Hi @johnnyjong823, Thank you for your contribution. The PR has been merged (#5).
